config.yml

```diff
@@ -95,6 +95,9 @@ defaults:
     # Trust a forwarding proxy
     trustProxy:               !env:bool TRUST_PROXY
 
+    # TODO: pick a better name for this
+    proxyName:                !env PROXY_NAME
```

Comment:
May I suggest `pathPrefix` or something else? maybe `vhost` (probably bad).

If you want to leave this with a deliberately ugly name, please add a comment explaining what it is..


----
I think this often called `virtualPath`

Reply:
I think if we do this right, we can actually _simplify_ this whole process and perhaps even make the `/v1` bit useful all at one fell swoop.

I also notice that at least in auth, `publicUrl` does pretty much nothing other than be in schema IDs. I'm thinking that maybe we can get rid of it and just do relative paths for that too.